x-pack/plugins/security_solution/public/detections/components/host_isolation/take_action_dropdown.tsx

```diff
@@ -7,45 +7,49 @@
 
 import React, { useState, useCallback, useMemo } from 'react';
 import { EuiContextMenuItem, EuiContextMenuPanel, EuiButton, EuiPopover } from '@elastic/eui';
-import { ISOLATE_HOST } from './translations';
+import { ISOLATE_HOST, UNISOLATE_HOST } from './translations';
 import { TAKE_ACTION } from '../alerts_table/alerts_utility_bar/translations';
+import { useHostIsolationStatus } from '../../containers/detection_engine/alerts/use_host_isolation_status';
 
 export const TakeActionDropdown = React.memo(
-  ({ onChange }: { onChange: (action: 'isolateHost') => void }) => {
+  ({
+    onChange,
+    agentId,
+  }: {
+    onChange: (action: 'isolateHost' | 'unisolateHost') => void;
+    agentId: string;
```

Comment:
Is this the Fleet `agentId` or the Endpoint ID?

I think we talked about this just a moment ago, where maybe the check to see if the agent/host is isolate is done outside of this component and instead drive which of the menu options should be visible via a prop (ex. `isCurrentlyIsolated: boolean`

Reply:
i believe it is the `agentId`. and i can move the check outside.